src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/SigninController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers
+{
+    [Authorize]
+    [ApiExplorerSettings(IgnoreApi=true)]
+    public class SigninController : Controller
+    {
+        // GET
+        public IActionResult Index()
+        {
+            return Redirect("/admin-dashboard");
```

Comment:
Nice!

Reply:
Not ideal but it will give us something to work with quickly for now